src/Digdir.Domain.Dialogporten.Infrastructure/InfrastructureExtensions.cs

```diff
@@ -74,15 +74,19 @@ public static class InfrastructureExtensions
 
         services.ConfigureFusionCache(nameof(Altinn.NameRegistry), new()
         {
-            Duration = TimeSpan.FromDays(1)
+            Duration = TimeSpan.FromHours(24),
+            FailSafeMaxDuration = TimeSpan.FromHours(26)
         })
         .ConfigureFusionCache(nameof(Altinn.ResourceRegistry), new()
         {
-            Duration = TimeSpan.FromMinutes(20)
+            Duration = TimeSpan.FromMinutes(20),
+            // The resource list is several megabytes and might take a while to process
+            FactoryHardTimeout = TimeSpan.FromSeconds(10)
         })
         .ConfigureFusionCache(nameof(Altinn.OrganizationRegistry), new()
         {
-            Duration = TimeSpan.FromDays(1)
+            Duration = TimeSpan.FromHours(24),
+            FailSafeMaxDuration = TimeSpan.FromHours(26)
```

Comment:
(off-topic) Added to avoid Fusion cache complaining about Duration being larger than FailSafeMaxDuration